### ISSUE TITLE
Documentation Issues - 2018

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6717]
Issue overview
--------------
- [x] I/O Ref for HeatExchanger:FluidToFluid should clearly state that is does not work with steam loops.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Ticket added to Pivotal for defect (development team task)
 - [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
- [x] Engineering Reference, should say "one or more . . . reference points".  Check for any other spots that may mention only two reference points for split-flux.  DeLight is still limited to 100.  Ref. helpdesk ticket 12843.

> 7.3 Time-Step Daylighting Calculation
7.3.1 Overview
A daylighting calculation is performed each time step that the sun is up for each zone that has
one or two daylighting reference points specified.

**Comment by mjwitte** [comment]
- [x] For FaultModel:Fouling:* objects, in the Input Output Reference, the description of "Severity Schedule Name" is a pasteo from other fault models (talks about a temperature difference). This schedule is actually applied by taking the nominal Fouling Factor and *dividing* by the severity schedule value with a limit on the resulting fouling factor of max 1.0.  Also if the severity schedule value is zero, then the nominal Fouling Factor is used.  Code for this is [here](https://github.com/NREL/EnergyPlus/blob/836ce9e0b62e671b78718bb0a5a5e586b3e38559/src/EnergyPlus/FaultsManager.cc#L1893-L1905).

Also, the description for Fouling Factor (for FaultModel:Fouling:Chiller) should explain that the fouling factor is applied to the nominal chiller capacity *and* COP, as described in the Engineering Reference.  Should check this for the other FaultModel:Fouling:* equipment types and make sure it's clear in the I/O Ref how the fouling factor gets used.. 

Ref [Unmethours 32011](https://unmethours.com/question/32011/energy-plus-how-to-build-a-faultmodelfoulingchiller/)

**Comment by eringold** [comment]
- [x] AvailabilityManager:HybridVentilation field 'Ventilation Control Mode Schedule Name' integer list not showing up in compiled documentation ([here](https://bigladdersoftware.com/epx/docs/8-9/input-output-reference/group-system-availability-managers.html#field-ventilation-control-mode-schedule-name)). However, the list can be found in the [documentation source tex](https://github.com/NREL/EnergyPlus/blob/dc4f43d7dee80127b5545a0beb0a58c74c46f199/doc/input-output-reference/src/overview/group-system-availability-managers.tex#L554-L592). 

@michael-okeefe

**Comment by michael-okeefe** [comment]
@eringold Thank you for the notification flag to make me aware of the issue on the HTML documentation side.

I believe I have a fix for this and will regenerate the web documentation as soon as I can get to it. Thank you.

**Comment by michael-okeefe** [comment]
@eringold I fixed the issue in the live web documentation (be sure to refresh your browser if it doesn't immediately show up). I've checked the check box in your report to close it out. Thanks again for bringing it to our attention.

**Comment by mjwitte** [comment]
- [x] Input Output Reference, typo in PlantComponent:UserDefined, after Number of Plant Loop Connections, next heading starts with "FeildSet:".

**Comment by eringold** [comment]
- [x] Input Output Reference, typo in Daylighting:ReferencePoint: "Guidelines for Daylighting Modeling Ssing the SplitFlux Method": https://github.com/NREL/EnergyPlus/blob/13627153888d3ac71645178a674491bc8151b8b6/doc/input-output-reference/src/overview/group-daylighting.tex#L215

**Comment by shorowit** [comment]
Issues noticed with [EvaporativeCooler:Direct:ResearchSpecial](https://bigladdersoftware.com/epx/docs/8-9/input-output-reference/group-evaporative-coolers.html#evaporativecoolerdirectresearchspecial):

- [x] Field: Recirculationg Water Pump Design Power -- typo
- [x] Two fields (Recirculating Water Pump Design Power and Water Pump Power Sizing Factor) have references to `Secondary Design Air Flow Rate`, which is related to the EvaporativeCooler:Indirect:ResearchSpecial object, not this object. Should presumably say Primary here instead of Secondary.
- [x] The example IDF entry is missing the field Primary Design Air Flow Rate.

**Comment by michael-okeefe** [comment]
Just noticed this one in [Output Details and Examples / eplustbl.&lt;ext&gt;](https://bigladdersoftware.com/epx/docs/8-9/output-details-and-examples/eplustbl-lt-ext-gt.html#utilitycosttariff):

- [x] False statement: "The use of UtilityCost:Tariff objects automatically generates two reports related to the calculation of annual utility costs" but according to [this unmet hours issue](https://unmethours.com/question/22123/no-report-for-tariffs/?answer=22135#post-id-22135) that is not the case.

**Comment by Nigusse** [comment]
- [x] In V8.9 I/O reference document section 3.2 Introduction to Economics, the sentence "Future versions of EnergyPlus will add other investment related analyses such as life-cycle cost" [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/input-output-reference/src/energyplus-economics/introduction-to-economics.tex#L26) seems outdated. We already have life cycle cost estimation capability.

- [x] In V8.9 I/O reference document section 3.4.1.0.9 Field: Demand Window Length, the word "Instead" is duplicate in this sentence "If the Timestep is 6, i.e. 10 minute timesteps, then QuarterHour is not used and instead HalfHour is used instead." [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/input-output-reference/src/energyplus-economics/utilitycost-tariff.tex#L326).

**Comment by mjwitte** [comment]
- [x] In the EMS Application Guide, under Actuators, HVAC Systems, Pump, the "Pump Pressure Rise" actuator is not listed. Here's the actuator setup code:
```
                SetupEMSActuator("Pump",
                                 PumpEquip(PumpNum).Name,
                                 "Pump Pressure Rise",
                                 "[Pa]",
                                 PumpEquip(PumpNum).EMSPressureOverrideOn,
                                 PumpEquip(PumpNum).EMSPressureOverrideValue);
 ```
And the override value is used here:
```
Power = VolFlowRate * PumpEquip(PumpNum).EMSPressureOverrideValue / TotalEffic
```

**Comment by shorowit** [comment]
- [x] Engineering Reference Figure 124 describes the PlantLoop's UniformLoad scheme as "Load all equipment uniformly with equal load to capacity", which appears to be contradicted by the paragraph below it that says "The UNIFORMLOAD scheme first divides the load evenly among all available components. If some components do not have the capacity to meet the uniformly distributed load, the remaining load is divided among the remaining components." 

For a building with two pieces of equipment, where Equipment A has twice the capacity of Equipment B, and UniformLoad is used as the Load Distribution Scheme, do both pieces of equipment see the same load or does Equipment A see twice the load of Equipment B?

**Comment by mjwitte** [comment]
@shorowit The two statements are attempting to say the same thing, obviously not very clearly - "to capacity" is trying to say what the paragraph explains in more words.
Probably need to prove this with a test file, but here's my understanding of UniformLoad;
Let's say Equipment A has capacity of 100, and Equipment B has capacity of 50.
If the current load is 60, theN both will be loaded at 30.
If the current load is 130, then uniform load says each one should take 65, but Equipment B can only take 50, so Equipment A will be given 65+15=80.
Maybe an example table with a scenario like this showing how the equipment would be loaded with each loading scheme would be helpful here.

**Comment by shorowit** [comment]
@mjwitte Yes, some additional clarification would be helpful. I would not have arrived at that logic from the descriptions. The phrasing "equal load to capacity" suggests to me that if Equipment A has capacity of 100 and Equipment B has capacity of 50, and the current load is 60, then Equipment A would be loaded at 40 and Equipment B would be loaded at 20.

**Comment by aaron-boranian** [comment]
- [x] I/O Ref section for `Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed` object is missing. It is referenced in the [Heat Pump Water Heaters section](https://github.com/NREL/EnergyPlus/blob/eb68ccb2bfb3d01807412e33399299da8cdb4b11/doc/input-output-reference/src/overview/group-water-heaters.tex#L1161) of the I/O Ref, but it's input fields are not discussed or defined like the `Coil:WaterHeating:AirToWaterHeatPump:Pumped` and `Coil:WaterHeating:AirToWaterHeatPump:Wrapped `objects are in the Heating and Cooling Coils section defined by `doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex`. 

**Edit:** 

* [x] Noticed that this object is listed as `Coil:WaterHeating:AirToWaterHeatPump:--VariableSpeed` in two files:
- [`doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex`](https://github.com/NREL/EnergyPlus/blob/eb68ccb2bfb3d01807412e33399299da8cdb4b11/doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex#L5197)
- [`doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex`](https://github.com/NREL/EnergyPlus/blob/f9f6a3f0a41076e9e2e0675ac3e8cd15216a397b/doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex#L5294)

Maybe a typo? Either way, the description of the variable speed WSHP coil object's input fields are still missing.

**Comment by aaron-boranian** [comment]
- [x] I/O Ref and Eng Ref sections for `FaultModel:PressureSensorOffset:OutdoorAir` object are missing. Content should be added to:
- [`doc/input-output-reference/src/overview/group-operational-faults.tex`](https://github.com/NREL/EnergyPlus/blob/eb68ccb2bfb3d01807412e33399299da8cdb4b11/doc/input-output-reference/src/overview/group-operational-faults.tex)
- [`doc/engineering-reference/src/operational-faults.tex`](https://github.com/NREL/EnergyPlus/blob/f9f6a3f0a41076e9e2e0675ac3e8cd15216a397b/doc/engineering-reference/src/operational-faults.tex)

**Comment by aaron-boranian** [comment]
- [x] I/O Ref and Eng Ref sections for `Curve:ChillerPartLoadWithLift` object are missing. It is referenced in the [Plant Equipment section](https://github.com/NREL/EnergyPlus/blob/develop/doc/input-output-reference/src/overview/group-plant-equipment.tex#L1842) of the I/O Ref and [Chillers section](https://github.com/NREL/EnergyPlus/blob/develop/doc/engineering-reference/src/simulation-models-encyclopedic-reference/chillers.tex#L1754) of the Eng Ref, but it's input fields are not discussed or defined like the `Curve:ChillerPartLoadCustom` object is in the Performance Curves section defined by `doc/input-output-reference/src/overview/group-performance-curves.tex`.  Maybe  `Curve:ChillerPartLoadWithLift` (appears in the IDD) and `Curve:ChillerPartLoadCustom` (**does not** appear in the IDD) are the same? If so, `Curve:ChillerPartLoadWithLift*Curves*` appears in the [`Chiller:Electric:ReformulatedEIR` section of the IDD](https://github.com/NREL/EnergyPlus/blob/develop/idd/Energy+.idd.in#L68493) and the first I/O Ref link of this post, which may be a typo.

**Comment by jmarrec** [comment]
@aaron-boranian fpr "I/O Ref and Eng Ref sections for FaultModel:PressureSensorOffset:OutdoorAir object are missing", that's because this object, while present in the EnergyPlus Input Data Dictionary, is not implemented yet. See https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/FaultsManager.cc#L187

Should we keep not mentioning it, or add a section?

```latex
\subsection{FaultModel:PressureSensorOffset:OutdoorAir}%
\label{sub:faultmodel-pressuresensoroffset-outdoorair}

This object, while present in the EnergyPlus Input Data Dictionary, is not implemented yet.
```

**Comment by mjwitte** [comment]
@jmarrec @aaron-boranian Given that it's not implemented, that should have never been added to the IDD. Probably slipped in unnoticed with a larger group of new fault model objects. 

@Myoldmopar If you agree I will remove this object from the IDD in #6951. There are some comments in the code that mention this - remove those as well?

**Comment by jmarrec** [comment]
@mjwitte @shorowit I tried to add some info regarding the Plant Load Distribution Scheme. I only did "Sequential Load" and "UniformLoad" since it's extremely painful to create, and I think having two would make people understand the three others... Could you let me know if this PDF extract could address your issue please?

[test.pdf](https://github.com/NREL/EnergyPlus/files/2496481/test.pdf)

**Comment by jmarrec** [comment]
I think the last other checkbox from Aaron: 

" I/O Ref section for Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object is missing. It is referenced in the Heat Pump Water Heaters section of the I/O Ref, but it's input fields are not discussed or defined like the Coil:WaterHeating:AirToWaterHeatPump:Pumped and Coil:WaterHeating:AirToWaterHeatPump:Wrapped objects are in the Heating and Cooling Coils section defined by doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex."

Should be made into its own issue, since that's going be quite some work (including tracking any original design docs, etc)

----

**Edit:** I ended up adding it in #7023 as well

**Comment by jmarrec** [comment]
Added an example for each "Load Distribution Scheme" while reading the source code in `DistributePlantLoad`
See [test.pdf](https://github.com/NREL/EnergyPlus/files/2502185/test.pdf)

**Comment by eringold** [comment]
@michael-okeefe - another issue on the Big Ladder Software documentation site. 

In the v9.0 Engineering Reference, a link to the 'Coils' Encyclopedic Reference [section](https://bigladdersoftware.com/epx/docs/9-0/engineering-reference/coils.html) is missing from the [table of contents](https://bigladdersoftware.com/epx/docs/9-0/engineering-reference/).

**Comment by michael-okeefe** [comment]
Thank you, @eringold . We'll look into it and report back.

**Comment by Myoldmopar** [comment]
> @Myoldmopar If you agree I will remove this object from the IDD in #6951. There are some comments in the code that mention this - remove those as well?

It looks like you ended up *not* removing that object from the IDD.  I'm OK with that, I just want to confirm that was your intent.  Otherwise the PR looks great and I'd like to merge it down.

**Comment by mjwitte** [comment]
@Myoldmopar I did not remove it yet, because I wanted confirmation to go ahead. I can add that over in #6951, or you we can remove it here.

**Comment by Myoldmopar** [comment]
Add it over there, I'm going to merge this one.

**Comment by mjwitte** [comment]
Done, see https://github.com/NREL/EnergyPlus/pull/6951/commits/1171302bf0ccb8775b4a5824f69040f3a10337df

**Comment by Myoldmopar** [comment]
Closed via #7023
